### PR TITLE
multi type and documentation hash

### PR DESCRIPTION
**Pull request by scauglog**
multi type parameter default to first type
when :type and :default are provided in a documenation hash they override grape :type and :default

### CODE REVIEW

**Comment by scauglog** [comment]
resolve issue #443

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,11 +2,12 @@
 
 #### Features
 
-* Your contribution here.
+* when a parameter as multi types the first type is use as the documentation type
```
Comment (LeFnord) [inline CHANGELOG.md]: please could adept it to the normal changeling format, thanks

**Comment by LeFnord** [comment]
👍 … thanks, good work, please run `bundle exec rake`, so you see the rubocop hints 😉

**Comment by scauglog** [comment]
sorry

**Comment by LeFnord** [comment]
please adept the travis file, this feature is only available for grape >= 0.14.0

**Comment by scauglog** [comment]
I run the multi type test only `if Grape::VERSION >= '0.14'`. Is that good ?

**Comment by LeFnord** [comment]
@scauglog … nice solution ≥ and if a user tries to uses `types` but has only grape 0.13.0, then it raises an error, right?
I'm fine with it …

**Comment by scauglog** [comment]
yes, grape will raise an `unknown validator: types`. The same error that make the test fail for grape 0.13 and 0.12. The problem was that before grape 0.14 you can't use `types` when defining a parameter. I do some research and before grape 0.14 you can use `type` with an array of type but grape will take the first element of the array to check type (and I set the swagger type to the first element of the array). 

so now I test `type: [String, Integer]` for all grape version and `types: [String, Integer]` if grape >= 0.14. All case I can think are tested if you see anything else tell me.

**Comment by LeFnord** [comment]
@scauglog … thanks 👍

**Pull request merged**